Plugin Store StoreExt ctx doesnt exist

The below example seems to be no longer valid in V2. Please update V2 docs when a plugin has changed it's functionality. (See https://github.com/tauri-apps/tauri-plugin-store/tree/v2 for partially updated docs)

```rust
use tauri::Wry;
use tauri_plugin_store::StoreExt;
use serde_json::json;

#[cfg_attr(mobile, tauri::mobile_entry_point)]
pub fn run() {
    tauri::Builder::default()
        .plugin(tauri_plugin_store::Builder::default().build())
        .setup(|app| {
            let store = app.handle().store_builder("store.bin").build();

            // Note that values must be serde_json::Value instances,
            // otherwise, they will not be compatible with the JavaScript bindings.
            store.insert("some-key", json!({ "value": 5 }))?;

            // Get a value from the store.
            let value = store.get("some-key").expect("Failed to get value from store");
            println!("{}", value); // {"value":5}

            // You can manually save the store after making changes.
            // Otherwise, it will save upon graceful exit as described above.
            store.save()?;


            Ok(())
        })
        .run(tauri::generate_context!())
        .expect("error while running tauri application");
}
```

(Primary plugin docs here: https://v2.tauri.app/plugin/store/)

```
    let mut defaults = std::collections::HashMap::new();
    defaults.insert(DISPLAY_CONFIG.to_string(), json!(get_display_config()));
    defaults.insert(SWIPER_CONFIG.to_string(), json!(get_swiper_config()));

    let store = app
        .store_builder("store.bin")
        .defaults(defaults)
        .auto_save(Duration::from_millis(100))
        .build();

    let _ = store.load().expect("file is not existed");
```
panic:  file is not existed: Io(Os { code: 2, kind: NotFound, message: "没有那个文件或目录" })
i don't know how to fix it .

--- a/src/content/docs/plugin/store.mdx
+++ b/src/content/docs/plugin/store.mdx
@@ -80,8 +80,8 @@ import { load } from '@tauri-apps/plugin-store';
 // when using `"withGlobalTauri": true`, you may use
 // const { load } = window.__TAURI__.store;
 
-// create a new store or load the existing one
-const store = await load('store.bin');
+// Create a new store or load the existing one, note that the options will be ignored if a `Store` with that path has already been created
+const store = await load('store.bin', { autoSave: false });
 
 // Set a value.
 await store.set('some-key', { value: 5 });
@@ -91,7 +91,9 @@ const val = await store.get<{ value: number }>('some-key');
 console.log(val); // { value: 5 }
 
 // You can manually save the store after making changes.
-// Otherwise, it will save upon graceful exit as described above.
+// Otherwise, it will save upon graceful exit
+// And if you set `autoSave` to a number or left empty,
+// it will save the changes to disk after a debounce delay, 100ms by default.
 await store.save();
 ```
 
@@ -108,6 +110,9 @@ pub fn run() {
     tauri::Builder::default()
         .plugin(tauri_plugin_store::Builder::default().build())
         .setup(|app| {
+            // Create a new store or load the existing one
+            // this also put the store in the app's resource table
+            // so your following calls `store` calls (from both rust and js) will reuse the same store
             let store = app.store("store.json")?;
 
             // Note that values must be serde_json::Value instances,
@@ -117,6 +122,9 @@ pub fn run() {
             // Get a value from the store.
             let value = store.get("some-key").expect("Failed to get value from store");
             println!("{}", value); // {"value":5}
+
+            // Remove the store from the resource table
+            store.close_resource();
 
             Ok(())
         })
@@ -128,8 +136,17 @@ pub fn run() {
 </TabItem>
 </Tabs>
 
-## Migrating from v1 and v2 beta/rc
+### LazyStore
 
+There's also a high level JavaScript API `LazyStore` which only loads the store on first access
+
+```typescript
+import { LazyStore } from '@tauri-apps/plugin-store'
+
+const store = new LazyStore('settings.json')
+```
+
+## Migrating from v1 and v2 beta/rc
 
 <Tabs>
 <TabItem label="JavaScript">

--- a/src/content/docs/plugin/store.mdx
+++ b/src/content/docs/plugin/store.mdx
@@ -76,15 +76,12 @@ Install the store plugin to get started.
 <TabItem label="JavaScript">
 
 ```typescript
-import { createStore } from '@tauri-apps/plugin-store';
+import { load } from '@tauri-apps/plugin-store';
 // when using `"withGlobalTauri": true`, you may use
-// const { createStore } = window.__TAURI__.store;
+// const { load } = window.__TAURI__.store;
 
 // create a new store or load the existing one
-const store = await createStore('store.bin', {
-  // we can save automatically after each store modification
-  autoSave: true,
-});
+const store = await load('store.bin');
 
 // Set a value.
 await store.set('some-key', { value: 5 });
@@ -111,7 +108,7 @@ pub fn run() {
     tauri::Builder::default()
         .plugin(tauri_plugin_store::Builder::default().build())
         .setup(|app| {
-            let store = app.handle().store_builder("store.bin").build();
+            let store = app.store("store.json")?;
 
             // Note that values must be serde_json::Value instances,
             // otherwise, they will not be compatible with the JavaScript bindings.
@@ -121,16 +118,37 @@ pub fn run() {
             let value = store.get("some-key").expect("Failed to get value from store");
             println!("{}", value); // {"value":5}
 
-            // You can manually save the store after making changes.
-            // Otherwise, it will save upon graceful exit as described above.
-            store.save()?;
-
-
             Ok(())
         })
         .run(tauri::generate_context!())
         .expect("error while running tauri application");
 }
+```
+
+</TabItem>
+</Tabs>
+
+## Migrating from v1 and v2 beta/rc
+
+
+<Tabs>
+<TabItem label="JavaScript">
+
+```diff
+- import { Store } from '@tauri-apps/plugin-store';
++ import { LazyStore } from '@tauri-apps/plugin-store';
+```
+
+</TabItem>
+<TabItem label="Rust">
+
+```diff
+- with_store(app.handle().clone(), stores, path, |store| {
+-     store.insert("some-key".to_string(), json!({ "value": 5 }))?;
+-     Ok(())
+- });
++ let store = app.store(path)?;
++ store.set("some-key".to_string(), json!({ "value": 5 }));
 ```
 
 </TabItem>

--- a/src/content/docs/plugin/store.mdx
+++ b/src/content/docs/plugin/store.mdx
@@ -143,9 +143,9 @@ pub fn run() {
 There's also a high level JavaScript API `LazyStore` which only loads the store on first access
 
 ```typescript
-import { LazyStore } from '@tauri-apps/plugin-store'
+import { LazyStore } from '@tauri-apps/plugin-store';
 
-const store = new LazyStore('settings.json')
+const store = new LazyStore('settings.json');
 ```
 
 ## Migrating from v1 and v2 beta/rc

--- a/src/content/docs/plugin/store.mdx
+++ b/src/content/docs/plugin/store.mdx
@@ -82,7 +82,7 @@ import { load } from '@tauri-apps/plugin-store';
 
 // Create a new store or load the existing one,
 // note that the options will be ignored if a `Store` with that path has already been created
-const store = await load('store.bin', { autoSave: false });
+const store = await load('store.json', { autoSave: false });
 
 // Set a value.
 await store.set('some-key', { value: 5 });

--- a/src/content/docs/plugin/store.mdx
+++ b/src/content/docs/plugin/store.mdx
@@ -80,7 +80,8 @@ import { load } from '@tauri-apps/plugin-store';
 // when using `"withGlobalTauri": true`, you may use
 // const { load } = window.__TAURI__.store;
 
-// Create a new store or load the existing one, note that the options will be ignored if a `Store` with that path has already been created
+// Create a new store or load the existing one,
+// note that the options will be ignored if a `Store` with that path has already been created
 const store = await load('store.bin', { autoSave: false });
 
 // Set a value.
@@ -112,7 +113,8 @@ pub fn run() {
         .setup(|app| {
             // Create a new store or load the existing one
             // this also put the store in the app's resource table
-            // so your following calls `store` calls (from both rust and js) will reuse the same store
+            // so your following calls `store` calls (from both rust and js)
+            // will reuse the same store
             let store = app.store("store.json")?;
 
             // Note that values must be serde_json::Value instances,

